src/components/BookForm/BookForm.js

```diff
@@ -6,77 +6,187 @@ import Row from "react-bootstrap/Row";
     <Form>
       <h2 className="head">Book Form</h2>
       <Row className="m-2">
         <Form.Group as={Col} controlId="formGridBookName">
           <Form.Label>Название книги</Form.Label>
-          <Form.Control type="bookname" placeholder="Кобзарь" />
+          <Form.Control
+            value={book.bookName}
+            onChange={onChangeForm}
+            name="bookName"
+            placeholder="Кобзарь"
+          />
         </Form.Group>
 
         <Form.Group as={Col} controlId="formGridAuthorName">
           <Form.Label>Автор</Form.Label>
-          <Form.Control type="AuthorName" placeholder="Тарас Шевченко" />
+          <Form.Control
+            value={book.authorName}
+            onChange={onChangeForm}
+            name="authorName"
+            placeholder="Тарас Шевченко"
+          />
         </Form.Group>
       </Row>
 
       <Form.Group className="m-3" controlId="formGrid">
         <Form.Label>Описание</Form.Label>
-        <Form.Control as="textarea" rows={2} />
+        <Form.Control
+          value={book.discription}
+          name="discription"
+          onChange={onChangeForm}
+          as="textarea"
+          rows={2}
+        />
       </Form.Group>
 
-      {["Фантастика", "Детектив", "Драма"].map((label) => (
+      {CHECKBOXES.map(({ id }) => (
         <Form.Check
+          value={book.genre}
+          name="genre"
+          key={id}
           className="m-3"
           inline
-          label={label}
+          label={id}
           type="checkbox"
-          id={`inline-${label}-3`}
+          id={`inline-${id}-3`}
+          onChange={() => onChangeCheckbox(id)}
+          checked={book.genre.indexOf(id) > -1}
         />
       ))}
+      <Button onClick={() => checkAll()}>Чекбокс</Button>
 
       <Row className="m-2">
         <Form.Group as={Col} controlId="formGridCost">
           <Form.Label>Стоимость</Form.Label>
-          <Form.Control type="number" />
+          <Form.Control
+            value={book.cost}
+            name="cost"
+            onChange={onChangeForm}
+            type="number"
+          />
         </Form.Group>
 
         <Form.Group as={Col} controlId="formGridLanguage">
-          <Form.Label htmlFor="disabledSelect">Язык</Form.Label>
-          <Form.Select id="disabledSelect">
+          <Form.Label>Язык</Form.Label>
+          <Form.Select
+            value={book.language}
+            name="language"
+            onChange={onChangeForm}
+            id="disabledSelect"
+          >
+            <option disabled></option>
             <option>Украинский</option>
             <option>Английский</option>
           </Form.Select>
         </Form.Group>
 
         <div className="m-2">Наличие</div>
         <div className="availability">
-          {["Есть", "Нету"].map((label) => (
-            <div
-              key={`inline-${label}`}
-              className="m-1
-          "
-            >
+          {["Есть", "Нету"].map((availability) => (
```

Comment:
выведи массив данных в константу и положи в отдельный файл
и в массиве этом храни объекты для таких компонентов с разными полями
```suggestion
          const AVAILABILITIES= [
           {
            label: "Есть",
            value: true // или можно тот же текст, по твоему желанию
           },
             {
            label: "Нету",
            value: false // или можно тот же текст, по твоему желанию
           }
          ];
```